src/toast/close-button/ToastItemContext.tsx

```diff
@@ -0,0 +1,13 @@
+import {createContext} from "react";
```

Comment:
It doesn't make sense to have this file under `close-button` directory. Let's move it to `src/toast/` or `src/toast/util/`

Reply:
You're right. `src/toast` is better.